api/src/functions/root/api-upload-csv/csvHeaders.ts

```diff
@@ -0,0 +1,73 @@
+export const csvHeaders:Array<string> = new Array(
```

Comment:
We will need an endpoint for users to download a template csv file with just the headers in it. Can you create this and call it GET /api/myanmar/disasterAssessment/template. It should return the response with an HTTP header `Content-Type: text/csv`.

Reply:
This is complete. This function returns one additional header i.e. `Content-Disposition` so that the browser knows that the file needs to be saved instead of opened in a new tab.